src/apyfloatarray_wrapper.cc

```diff
@@ -522,7 +522,232 @@ void bind_float_array(nb::module_& m)
             &APyFloatArray::squeeze,
             nb::arg("axis") = nb::none(),
             R"pbdoc(
-            Squeeze the matrix, removing dimensions of size 1.
+            Removes dimensions of size 1 at the specified axis/axes, if no axís is given removes all dimensions with size 1.
+
+            Parameters
+            ----------
+            axis : tuple, int, optional
+                The axis/axes to squeeze, a given axis with a size other than 1 will result in an error. No given axes  will be remove all dimensions of size 1.
+
+            Returns
+            -------
+            :class:`APyFloatArray`
+
+            Raises
+            -------
+            ValueError
+                If given an axis of a size other than 1 a ValueError will be thrown.
+            IndexError
+                If a specified axis is outside of the existing number of dimensions for the array.
+
+            )pbdoc"
+        )
+
+        .def(
+            "sum",
+            &APyFloatArray::sum,
+            nb::arg("axis") = nb::none(),
+            R"pbdoc(
+            Returns the cumulative sum of the elements along specified axis/axes.
+
+            Parameters
+            ----------
+            axis : tuple, int, optional
+                The axis/axes to summate across. Will summate the whole array if no int or tuple is specified.
+
+            Returns
+            -------
+            :class:`APyFloatArray`
+
+            Raises
+            -------
+            IndexError
+                If a specified axis is outside of the existing number of dimensions for the array.
+
+            Examples
+            -------
+
+            >>> from apytypes import APyFloatArray
+
+            Array `a`, array to summate across.
+
+            >>> a = APyFloatArray(
+            ...     [1,2,3,4,5,6],
+            ...     exp_bits=5,
+            ...     man_bits=0
+            ... )
+
+            Array `b`, returned array where the specified axes are summated.
+            >>> b = a.sum()
+            In this case `b` will be equal to `c`.
```

Comment:
`b` has 5 exponent bits here while `c` has 8 exponent bits. Although they would be numerically equal it's clearer for the reader if the types has the same format.

Reply:
An error from my part, i copied the tests from APyFixedArray.sum() since they are almost the same and forget to update this